contrib/dtMediaWiki/dtMediaWiki.lua

```diff
@@ -0,0 +1,400 @@
+--[[dtMediaWiki is a darktable plugin which exports images to Wikimedia Commons
+    Author: Trougnouf (Benoit Brummer) <trougnouf@gmail.com>
+    Contributor: Simon Legner (simon04)
+
+Dependencies:
+* lua-sec: Lua bindings for OpenSSL library to provide TLS/SSL communication
+* lua-luajson: JSON parser/encoder for Lua
+* lua-multipart-post: HTTP Multipart Post helper
+]]
+local dt = require "darktable"
+local MediaWikiApi = require "contrib/dtMediaWiki/lib/mediawikiapi"
+local version = 47
+
```

Comment:
Translation would be nice.  examples/gettextExample.lua shows how

Reply:
Will do, but it might take a little while because I currently spend my free time getting a boat/radio/motorcycle license.

Reply:
@wpferguson done :-)

Reply:
Actually this crashes when I use the "_" function inside one of the functions registered in the dt.register_storage function, namely register_storage_initialize and register_storage_finalize (although adding a print(_()) statement in  register_storage_supported is an easy way to test).
Any idea what went wrong?